### ISSUE TITLE
Add Contact and Tel Field to ORG Object

### ISSUE THREAD

**Issue by martinhannigan** [peeringdb/peeringdb#1619]
**Is your feature request related to a problem? Please describe.**

An ORG can be owned by one party, but then have it's assets or operations managed by another. An ORG could also own a single FAC or NET object, and have different contact information for each. For example, it could be desirable for the ORG contact to be distinct from the operations contact.  One should also not assume sales@ is the desirable contact alias for all sales so enabling a unique ORG contact supports flexibility in other OBJ contact data.

**Who is affected by the problem?**

Small and large users of PeeringDB

**What is the impact?**

Continues to make peeringDB great.

**Are there security concerns?**

Doubtful.

**Are there privacy concerns?**

No, self entered data.

**Describe the solution you'd like**

1. Add an optional Contact: record for the ORG object allowing a name and email address to be displayed on the ORG object
2. Add an optional Telephone: record for the ORG object allowing the ORG to specify a telephone number to call the ORG directly.

**Do you think this feature will require a formal design?**
No

**Describe alternatives you've considered**

Continuing to not have streamlined contact information available representing how the business actually works. Not idea considering the proposed change should take little time and have small cost as well as there are available pixels on the ORG object UI.

**Could this feature request need support from the Admin Committee?**

Unlikely.

**What is the proposed priority?**

Normal.

**Comment by arnoldnipper** [comment]
Makes sense to rework the `poc` object to fit for `org`, `ix`, `net`, `carrier`, and `fac`

**Comment by jackcarrozzo** [comment]
PC: Seems reasonable - low priority but we should do it

**Comment by arnoldnipper** [comment]
Consensus is reached. However, an implementation design is needed, IMO

**Comment by leovegoda** [comment]
Yes. I'll schedule 20C to develop a design for the @peeringdb/pc to review

**Comment by grizz** [comment]
> Makes sense to rework the poc object to fit for org, ix, net, carrier, and fac

I'm all for this, which would accomplish adding those fields  to an org, however, I'm -1 for adding either field directly to an org.

I think we're on the same page here, I just wanted to make sure before we move into design.

Thanks @arnoldnipper for picking this up :)

**Comment by arnoldnipper** [comment]
> however, I'm -1 for adding either field directly to an org.

I'm not yet understanding what you mean here, @grizz. Afaik, we already have information which is inherited by daughter objects. I was thinking about the same for the new `poc` object. You can have a generic email, buzt also a specific email_net which then would only be shown at the `net` object.

**Comment by leovegoda** [comment]
Effort set to Minor for design work. Effort will be reviewed based on the design document.

**Comment by grizz** [comment]
> > however, I'm -1 for adding either field directly to an org.
> 
> I'm not yet understanding what you mean here, @grizz. Afaik, we already have information which is inherited by daughter objects. I was thinking about the same for the new `poc` object. You can have a generic email, but also a specific email_net which then would only be shown at the `net` object.

I guess I'm a data purist? I don't think it belongs there -- I won't fight it if everyone else thinks it should go there, but I do feel strongly enough to -1.

I think we should add a poc object to the org, move the email there, and then tel can be added there too.

**Comment by martinhannigan** [comment]
On Sun, Dec 8, 2024 at 18:28 Matt Griswold ***@***.***> wrote:

> however, I'm -1 for adding either field directly to an org.
>
> I'm not yet understanding what you mean here, @grizz
> <https://github.com/grizz>. Afaik, we already have information which is
> inherited by daughter objects. I was thinking about the same for the new
> poc object. You can have a generic email, but also a specific email_net
> which then would only be shown at the net object.
>
> I guess I'm a data purist? I don't think it belongs there -- I won't fight
> it if everyone else thinks it should go there, but I do feel strongly
> enough to -1.
>
> I think we should add a poc object to the org, move the email there, and
> then tel can be added there too.
>
Makes sense to me.

**Comment by leovegoda** [comment]
I'm getting a design developed for the PC to review.

**Comment by arnoldnipper** [comment]
> I think we should add a poc object to the org, move the email there, and then tel can be added there too.

@grizz, just to be sure. You are opting for a `poc` object for *every* object (i.e.  `carrier`, `fac`,  `ix`, `net`, and `org`)? Would be _pure_ from a DB pov, but not user-friendly, imo

**Comment by grizz** [comment]
Basically, although it could be exposed to the user in the same way either way of doing it.

**Comment by arnoldnipper** [comment]
> Basically, although it could be exposed to the user in the same way either way of doing it.

For the API as well, @grizz?

**Comment by grizz** [comment]
I was only referring to the API :)
